server/views/pages/licence/pdf-layout.njk

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+{% block head %}
+  <head>
+    <link href="{{ licencesUrl }}/assets/stylesheets/licence-pdf.css?{{ version }}" rel="stylesheet"/>
```

Comment:
Do you need {{licencesUrl}}? `layout.njk` doesnt have it for linking its stylesheets but maybe this is different because its sent to gotenberg

Reply:
Yes, its needed here - to reference the licence service from Gotenberg.